src/objects/Program_Enrollment__c.object

```diff
@@ -233,6 +233,20 @@
         <type>Text</type>
         <unique>false</unique>
     </fields>
+    <fields>
```

Comment:
Do we also want to add this field to the page layout? I can see going either way, since this is an optional feature.

Reply:
Good catch, I am not sure about this actually. @juliesnow and @aligeez, any idea on this? Should we display this field by default on program enrollment page layout?